spec/fixtures/custom_nginx.template

```diff
@@ -64,6 +64,10 @@ http {
         kong.init_worker()
     }
 
+    rewrite_by_lua_block {
+        kong.rewrite()
+    }
```

Comment:
We ought to run a diff on these 2 files one day because they might very will be less in sync that we think they are...